modules/cnv_migrate_vm_web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// None for CNV 1.4; topic saved for 2.0
+
+[[cnv-migrate-vm-web]]
+== Migrating a virtual machine
+
+Migrate a running virtual machine to a different node in the cluster. 
+
+Migrate a virtual machine using the &#8942; button of the template in the *Workloads* -> *Virtual Machines* list, or using the xref:cnv_vm_actions_web.adoc#cnv-vm-actions-web[*Actions* control] of the *Virtual Machine Details* screen.
```

Comment:
* There is no "Migrate ..." in xref:cnv_vm_actions_web.adoc#cnv-vm-actions-web[*Actions* control] documented
* There is no "Migrate... " available on the &#8942; button. 

@jelkosz is Migration described in this file possible at all?

Reply:
@alexxa Sorry, I resolved the conversation Tomas and I had about this file: Migrate is not possible, but it was when I wrote this up. I've taken this file out of the assembly so it won't be published. It seems as though the procedure will be the same so it made sense to include the file in the MR for when the feature becomes available.

Reply:
I would remove it from this MR, and keep it elsewhere then.
Also, [1] should be updated for Migrating feature accordingly.

[1] https://github.com/openshift/openshift-docs/blob/13ccc6480c4fc90aaeb207d19dc5dc657998914d/modules/cnv_vm_actions_web.adoc